client/resources/aria-at-test-io-format.mjs

```diff
@@ -1455,7 +1455,7 @@ export class TestRunInputOutput {
         output: command.atOutput.value,
         assertionResults: command.assertions.map(assertion => ({
           assertion: {
-            priority: assertion.priority === 1 ? 'MUST' : 'SHOULD',
+            priority: assertion.priority === 1 ? 'REQUIRED' : 'OPTIONAL',
```

Comment:
Was wondering why this was getting updated in this PR and realized this change was never captured in `w3c/aria-at` where these files should have been copied over from. So I've submitted https://github.com/w3c/aria-at/pull/1056 to make sure this change is captured for the future.

Reply:
Should that PR be merged first?

Reply:
Yep, I went ahead and merged!